samples/udpsrv.lua

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env lua
+
+--
+-- Sample UDP server. Listens on 3333 port
+-- Use netcat to send strings to server (ncat -u 127.0.0.1 3333)
+--
+
+local lgi = require 'lgi'
+local Gio = lgi.Gio
+
+local app = Gio.Application { application_id = 'org.v1993.udptest', flags = 'NON_UNIQUE' }
+
+local socket = lgi.Gio.Socket.new('IPV4', 'DATAGRAM', 'UDP')
```

Comment:
Use `Gio.SocketFamily.IPV4` rather than strings? (Same for DATAGRAM and UDP)

Reply:
Replaced